src/sentry/api/endpoints/notifications/notification_actions_index.py

```diff
@@ -21,9 +22,21 @@
 class NotificationActionsPermission(OrganizationPermission):
     scope_map = {
         "GET": ["org:read", "org:write", "org:admin"],
-        "POST": ["org:write", "org:admin"],
-        "PUT": ["org:write", "org:admin"],
-        "DELETE": ["org:write", "org:admin"],
+        "POST": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
+        "PUT": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
+        "DELETE": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
```

Comment:
If we're only updating the POST, we should leave the PUT and DELETE permission scopes the same

Reply:
the PUT and DELETE are used in `NotificationActionDetails`

Reply:
ok should be good then